src/status_im2/contexts/communities/menus/channel_options/view.cljs

```diff
@@ -0,0 +1,116 @@
+(ns status-im2.contexts.communities.menus.channel-options.view
+  (:require [utils.i18n :as i18n]
+            [utils.re-frame :as rf]
+            [quo2.core :as quo]))
+
+(defn hide-sheet-and-dispatch
+  [event]
+  (rf/dispatch [:bottom-sheet/hide])
+  (rf/dispatch event))
+
+(defn mute-channel-action
+  [chat-id]
+  (hide-sheet-and-dispatch [:chat.ui/mute chat-id true]))
+
+(defn unmute-channel-action
+  [chat-id]
+  (hide-sheet-and-dispatch [:chat.ui/mute chat-id false]))
+
+(defn view-members
+  [id]
+  {:icon                :i/members
+   :accessibility-label :view-members
+   :label               (i18n/label :t/view-channel-members-and-details)
+   :on-press            #()})
```

Comment:
Wouldn't it be better to not pass `on-press` if it does nothing? Or maybe add a `js/alert` at least.